powerauth-nextstep-client/src/main/java/io/getlime/security/powerauth/lib/nextstep/client/NextStepClient.java

```diff
@@ -199,6 +200,88 @@ private RestTemplate defaultTemplate() {
     }
 
     /**
+     * Get all authentication methods supported by Next Step server.
+     *
+     * @return List of authentication methods wrapped in GetAuthMethodResponse.
+     * @throws NextStepServiceException Exception with {@link ErrorModel} for ERROR status
+     */
+    public Response<GetAuthMethodsResponse> getAuthMethods() throws NextStepServiceException {
```

Comment:
We generally do not use GET methods in NextStep server, in favor of method like calls. I know this violates REST, RESTful, HATEOAS, and many more, but we pragmatically try to avoid keeping sensitive data in logs, which is desired in backend-to-backend communication with security related systems.

Please could you change the methods to POST and use request object in query?